README.md

```diff
@@ -1,2 +1,214 @@
 # bclconvert
-Workflow to process a run directory through bclconvert on hpc
+
+Workflow to produce FASTQ files from an Illumina instrument's run directory using DRAGEN bclconvert
+
+## Overview
+
+## Dependencies
+
+* [bclconvert](https://emea.support.illumina.com/sequencing/sequencing_software/bcl-convert.html)
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run bclconvert.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`runDirectory`|String|{'description': 'Illumina run directory (e.g. /path/to/191219_M00000_0001_000000000-ABCDE).', 'vidarr_type': 'directory'}
+`runName`|String|The name of the run, this will be used for the output folder and as a file prefix
+`samples`|Array[Sample]|array of Samples, that will includes names and barcodes
+`modules`|String|Modules to run on hpc
+
+
+#### Optional workflow parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+`lanes`|Array[Int]|[]|Extract reads only for specified lanes
+`basesMask`|String?|None|An Illumina bases mask string to use. If absent, the one written by the instrument will be used.
+`mismatches`|Int|1|Number of mismatches to allow in the barcodes (usually, 1)
+`timeout`|Int|40|The maximum number of hours this workflow can run for.
+
+
+#### Optional task parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+`runBclconvert.firstTileOnly`|Boolean|false|Flag for processing first tile only. Default false
+`runBclconvert.noLaneSplitting`|Boolean|false|Flag to disable lane splitting. Default false
+`runBclconvert.onlyMatchedReads`|Boolean|true|Process only matched reads. Default true
+`runBclconvert.fastqCompression`|String|'gzip'|Compression type of fastq files. Default gzip
+`runBclconvert.fastqCompressionLevel`|Int|1|Fastq compression level. Default 1
+`runBclconvert.memory`|Int|32|Memory allocated for running this task. Default 32
```

Comment:
"Default ___" at end of description redundant with Default column

Reply:
that is how readme generating script arranges things

Reply:
readme generating script pulls from the metadata in the workflow which could drop that part of the string, or the readme can be edited after the script generates it

Reply:
You can remove  "Default ___" from the parameter meta (e.g. https://github.com/oicr-gsi/bclconvert/pull/1/files#diff-b3a4ca45749e1ef8f3677f5beaf1da555d059b72cf9da42432eb47d32f2b27fdR188) and leave the generation of the default column/info to the readme generator.